extensions/amp-animation/0.1/web-animations.js

```diff
@@ -175,6 +190,44 @@ export class WebAnimationRunner {
   }
 
   /**
+   */
+  scrollTick(pos) {
```

Comment:
Ok for now. But I'm leaning toward all sub-animations should be of the same ticker as a rule and different tickers is more of an exception (i.e. maybe a separate animation). Topic for future discussion.